docs/src/layouts/base.astro

```diff
@@ -26,6 +26,13 @@ const { title } = Astro.props;
     />
     <title>{`${title === undefined ? "" : `${title} - `}Remeda`}</title>
     <script is:inline src="/dist/scripts/init-theme.js"></script>
+    <script type="module">
+      import * as R from "https://esm.sh/remeda";
+      window.R = R;
+      console.log(
+        "Use variable R right here to access the latest version of the library",
```

Comment:
I'm not sure about the wording here. I think it needs more context to make it obvious that this is an intended message and is here to guide the user. Without it, it looks like an error message. e.g., "Try out Remeda right here in your browser console; use the global `R` to access all the functions."

We can also utilize the fact that console.log can be styled to make the content bigger and more pronounced, at least using a header.

```js
console.log("%c<CONTENT>", "font-size: 24px; color: blue;")
```

@cjquines, wdyt?

Reply:
I'd appreciate better wording.

Actually the main purpose of this post for me is to highlight that the _latest_ version is being used, since R will also be available on the documentation page of the first version, but the examples will not work as expected.

Reply:
> console.log can be styled

Nice thought. I'd add (maybe) _italics_ and remove the color setting since the console background may vary.

Reply:
I'm OK with anything you find that looks nice and clear. I couldn't find examples of sites that do something similar, so we can't just copy. Please attach a screenshot once you update the PR so it's easy to review.

Reply:
> Actually the main purpose of this note for me is to highlight that the latest version is being used, since R will also be available on the documentation page of the first version, but the examples will not work as expected.

This is not a real concern of ours. We don't need to treat this feature as a sandbox for the current page, it should be seen as a site-wide feature. The v1 migration page is not an API, it's just a migration guide.

Reply:
<img width="650" alt="image" src="https://github.com/remeda/remeda/assets/15047511/749c8b6b-f2a5-434c-9336-0c0243f70c6e">
it's just that the non-monospace and increased font size seem to look neat.

Reply:
what about something like

```ts
console.log(
  "%cYou can use Remeda with the %cR%c global.",
  "font-family: sans-serif; font-size: 16px",
  "font-family: monospace; font-size: 16px",
  "font-family: sans-serif; font-size: 16px",
);
console.log(
  "%cTry typing %cR.add(5, 10)%c here:",
  "font-family: sans-serif; font-size: 16px",
  "font-family: monospace; font-size: 16px",
  "font-family: sans-serif; font-size: 16px",
);
```

Reply:
("here in your browser console" is, i think, redundant... if you're seeing this, you're already looking at the browser console?)

Reply:
I think this is enough to close the goal :)